tests/test_set_config.py

```diff
@@ -401,3 +396,94 @@ def test_values_needed_f90nml(): #pylint: disable=unused-variable
     salad.appetizer
 """
     assert result == outcome
+
+def test_cfg_to_yaml_conversion(): #pylint: disable=unused-variable
+    ''' Test that a .cfg file can be used to create a yaml object.'''
+    input_file = os.path.join(uwtools_file_base, pathlib.Path("fixtures/srw_example_yaml.cfg"))
+
+    with tempfile.TemporaryDirectory(dir='.') as tmp_dir:
+
+        out_file = f'{tmp_dir}/test_ouput.yaml'
+        args = ['-i', input_file, '-o', out_file, '--input_file_type', '.yaml']
+
+        set_config.create_config_obj(args)
+
+        expected = config.YAMLConfig(input_file)
```

Comment:
@j-derrico Nice find. The call to set_config is doing the dereferencing for you now. I think adding `expected.dereference_all()` here could do the trick.

Reply:
It looks like it's the out_file that's used in set_config.create-config_obj(args) that isn't being dereferenced. (The naming order for expected/rendered were accidentally flipped in compare_files().)